src/dblib.ml

```diff
@@ -38,12 +38,20 @@ let err_handler (f: severity -> int -> string -> unit) =
 let default_err_handler severity _err msg =
   raise(Error(severity, msg))
 
+let msg_handler (f: string -> severity -> int -> unit) =
+  Callback.register "Freetds.Dblib.msg_handler" f
+
+let default_msg_handler msg severity line =
+  let msg = sprintf "Error on line %d: %s" line msg in
+  raise(Error(severity, msg))
```

Comment:
Should the default handler raise `Error` by default?  (I don't remember if it can send informational messages only.)  In any case, the message should more clearly distinguish what comes from the `err_handler` and what comes from the `msg_handler`.

Reply:
I made it only raise on FATAL or CONSISTENCY by default, but I'm not sure what would make the documentation better. The distinction between the two is pretty arbitrary (from server, not from server), but they have different interfaces so I can't easily combine them.